opentext-page.html

```diff
@@ -0,0 +1,266 @@
+<!DOCTYPE html>
```

Comment:
opentext-page.html should not be in here...

You must have:
1. Worked on this opentext page on the branch you made for Open Text.
2. Switched to bootstrap branch, leaving #1 open.
3. Worked on opentext again at some point, forgetting you were on bootstrap branch.
4. Then you saved this opentext, effectively creating this page on the bootstrap branch.
5. Then when doing `git status` (if you did this command), you did not see that opentext was part of this, or if you did, you left it anyway as part of this commit, which is not correct.

This is an issue with knowing what branch you are on and correctly switching branches when you work on different pull requests...